src/tree/box.js

```diff
@@ -17,9 +18,9 @@ export default function box(divBox, o, appendNodes, cardStyling, reRender) {
     .attr('style', getStyle)
     .attr('id', d => d.data.id)
     .on('click', node => {
-      if (node.children) {
-        reRender(node.data.id);
+      if (node.data.id !== 'Root') {
```

Comment:
The old logic for traversing the tree will also be called from here. But this is fine. We will distinguish between them by looking at the target of the click event

Reply:
Ah okay. You are not creating a new div?

Reply:
I am creating a new div now :)